lib/sparql_to_sw_solr/instance_std_num_fields.rb

```diff
@@ -0,0 +1,108 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module InstanceStdNumFields
+
+      private
+
+      def add_std_num_fields(doc)
+        doc[:oclc] = oclc_nums
+        doc[:lccn] = lccn # single valued Solr field
+        doc[:isbn_display] = isbn_display_vals
+        doc
+      end
+
+      # see https://github.com/sul-dlss/solrmarc-sw/blob/master/stanford-sw/ ...
+      #     ... src/edu/stanford/StanfordIndexer.java#L792-#L836
+      #   we look for bf:Local identifiers with label OCoLC-M first
+      #   if none, we look for bf:Local identifiers with label OCoLC
+      #   we do not seek values prefixed 'ocn', 'ocm' or 'on' because the solrmarc algorithm seeks
+      #    those in 079, which are not converted.
+      def oclc_nums
+        ocolc_m = []
+        ocolc = []
+        oclc_solns.each do |soln|
+          # need if clauses for specs
+          value = soln.value.to_s.strip if soln && soln.bindings.keys.include?(:value)
+          label = soln.label.to_s.strip if soln && soln.bindings.keys.include?(:label)
```

Comment:
Why use `if soln`?  The code in this block iterates on a `soln`, so if there are none, the code in the block will not execute.  Can this be simplified to `if soln.bindings.keys.include?(:value)` and same for label?  Are the specs inserting `nil` into the ocln_solns array?  If there could be `nil` in that array, the iteration statement at the beginning of the block can exclude them using `Array#compact`, see https://apidock.com/ruby/v1_9_3_392/Array/compact